src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.health = 100;
+    this.name = name;
+    this.hidden = false;
```

Comment:
The 'hidden' property should not be part of every Animal instance, because it is specific to Herbivores only. Move this property to the Herbivore class.